### PR TITLE
Coinbase prime orders and wallets APIs

### PR DESCRIPTION
**Pull request by synerzipprasann**
@forus-coop/developers

## What's the Problem You're Solving?
Implement coinbase-prime node package to fetch coinbase prime APIs

## Fix
- Added all endpoints of Orders API
- Added all endpoints of Wallets API

## Checklist
- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [x] This PR only contains configuration changes
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then rebase)
- [x] My code follows the style guidelines, including naming conventions
- [x] I have commented my code with YARD docs
- [x] I have added explanatory comments, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by scottmuhs — CHANGES_REQUESTED** [review]
looking good, can we add some tests?

**Comment by synerzipprasann** [comment]
Hi @scottmuhs, I have added specs as you suggest, Could you please review it, when you get chance.

File: src/lib/activities.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Client } from "./client";
+import { formatResponse, ResponseFormat } from "./util";
+
+type listParams = {
```
Comment (scottmuhs) [inline src/lib/activities.ts:4]: Capitalized

File: src/lib/orders.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
+import { Client } from './client';
+import { formatResponse, ResponseFormat } from './util';
+
+type openOrdersParams = {
```
Comment (scottmuhs) [inline src/lib/orders.ts:4]: Caps

File: src/lib/orders.ts [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+  product_ids?: string,
+}
+
+type createParams = {
```
Comment (scottmuhs) [inline src/lib/orders.ts:8]: Caps

**Review by scottmuhs — APPROVED** [review]
Few Comments...can we name test failes as `fileName.test.ts` instead of spec.
Also all types/interfaces should be Capitalized

**Pull request merged**